tests/test_wave_01.py

```diff
@@ -120,10 +120,11 @@ def test_moves_movie_from_watchlist_to_empty_watched():
     assert len(updated_data["watched"]) is 1
     
     # *******************************************************************************************
-    # ****** Add assertions here to test that the correct movie was added to "watched" **********
+    assert updated_data["watched"][0] == {"title":MOVIE_TITLE_1, \
+        "genre":GENRE_1, "rating":RATING_1}
```

Comment:
Nice checking all the values of the dictionary! It was a good idea to split this line up due to length, but it would be even easier to read if we split it up into separate asserts:
```suggestion
    watched_movie = updated_data["watched"][0]
    assert watched_movie["title"] == MOVIE_TITLE_1
    assert watched_movie["genre"] == GENRE_1
    assert watched_movie["rating"] == RATING_1
```